test/test_circuit_builder.py

```diff
@@ -21,39 +21,27 @@ def test_simple(self) -> None:
             CNOT(0, 1),
         ]
 
-    def test_identity(self) -> None:
```

Comment:
What happened to these removed tests?

Reply:
It has been added to the `test_instructions` as part of the first test `"pauli gates"`.

Reply:
OK.